cmd/ethereum_rust/ethereum_rust.rs

```diff
@@ -137,6 +137,20 @@ async fn main() {
                     block.header.number, hash, error
                 );
             }
+            if store
+                .update_latest_block_number(block.header.number)
+                .is_err()
+            {
+                error!("Fatal: added block {} but could not update the block number -- aborting block import", block.header.number);
+                break;
+            };
+            if store
+                .set_canonical_block(block.header.number, hash)
+                .is_err()
+            {
+                error!("Fatal: added block {} but could not set it as canonical -- aborting block import", block.header.number);
+                break;
+            };
```

Comment:
This is already fixed by `apply_fork_choice`

Reply:
How so? 🤔

Reply:
Removed!